src/core/Toggle.tsx

```diff
@@ -4,22 +4,38 @@ import cn from "./utils/cn";
 
 type ToggleProps = {
   id: string;
+  size?: "sm" | "lg";
   label?: string;
   className?: string;
 } & React.ButtonHTMLAttributes<HTMLButtonElement>;
 
-const Toggle: React.FC<ToggleProps> = ({ id, label, className, ...props }) => {
+const Toggle: React.FC<ToggleProps> = ({
+  id,
+  size = "lg",
+  label,
+  className,
+  ...props
+}) => {
+  const rootSize = size === "sm" ? "w-[42px] h-[24px]" : "w-[56px] h-32";
```

Comment:
Afaik we have `h-24` out of the box already, so no need to have the custom style here